controllers/cartController.js

```diff
@@ -0,0 +1,20 @@
+const cartService = require("../services/cartService");
+
+const cartList = async(req, res) => {
+    try{
+        const {userId} = req.body;
+        if(!userId){
+            return res.status(400).json({message : "KEY_ERROR"});
+        };
+
+        const list = await cartService.cartList(userId);
```

Comment:
Service에 있는 함수도 마찬가지로 `cartList` -> `getCarts` 혹은 `getCartLists`로 수정해주세요.

```suggestion
        const list = await cartService.getCartList(userId);
```